pkg/setting/setting.go

```diff
@@ -988,6 +989,11 @@ func (cfg *Cfg) Load(args *CommandLineArgs) error {
 	cfg.PluginsEnableAlpha = pluginsSection.Key("enable_alpha").MustBool(false)
 	cfg.PluginsAppsSkipVerifyTLS = pluginsSection.Key("app_tls_skip_verify_insecure").MustBool(false)
 	cfg.PluginSettings = extractPluginSettings(iniFile.Sections())
+	pluginsAllowUnsigned := pluginsSection.Key("allow_loading_unsigned_plugins").MustString("")
```

Comment:
I think maybe we adding a new setting we must update the ini files? @marefr ?

Reply:
@kylebrandt Oh yeah, that's something I didn't think about.

Reply:
Yep, defaults.ini and sample.ini plus configuration.md